src/pages/reader/pdf.vue

```diff
@@ -1,33 +1,72 @@
 <template>
-  <div class="w-full h-full">
+  <div class="fixed inset-0 flex flex-col">
+    <header class="border-b border-[#ccc]">
+      <div class="flex items-center justify-between px-[16px] py-[8px]">
+        <div class="flex items-center gap-4">
+          <NuxtLink
+            :to="
+              localeLocation(
+                getAddress
+                  ? {
+                      name: 'id',
+                      params: { id: getAddress },
+                      query: { tab: 'collected' },
+                    }
+                  : { name: 'index' }
```

Comment:
to class page maybe?

Reply:
should change epub reader too?

Reply:
yes, merge this first if you want to do it in new pr